code/index.html

```diff
@@ -1,30 +1,96 @@
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Outfit:wght@300;700&display=swap" rel="stylesheet">  
+    <title>Fresh veggies home deliver</title>
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <!-- Tried to use BEM naming http://getbem.com/naming/ -->
+    <!-- Header with big hero background image -->
+    <header>
+      <h1>We deliver fresh veggies straight to your door</h1>
+    </header>
+    <!-- First section with short info text -->
+    <section class="info-section">
+      <h2 class="block-heading">
+        We are your local vegetable farmer. We offer two different subscription alternatives for you. Sign up with the form below.
+      </p>
+    </section>
+    <!-- Second section with two parts: A form and a FAQ accordion. Used Flexbox here to divide them and make it responsive. -->
+    <section class="row">
+      <!-- Form -->
+      <div class="form column">
+        <h2 class="block-heading">
+          Sign up and get fresh vegetables all summer!
+        </h2>
+        <p class="block-text">Where do you want your vegetables delivered?</p>
+        <form action="https://httpbin.org/anything" method="post">
+          <input type="text" required name="fullname" placeholder="Full name">
+          <input type="text" required name="address" placeholder="Delivery address">
+          <input type="tel" required name="telephone" placeholder="Telephone">
+          <input type="email" required name="email" placeholder="Email">
+          <p class="block-text form-text">Please choose your subscription plan</p>
+          <div class="form-group">
+            <label for="weekly">Weekly</label>
+            <input type="radio" id="weekly" class="form-radios" name="subscription_plan" value="Weekly">
+            <label for="monthly">Monthly</label>
+            <input type="radio" id="monthly" class="form-radios" name="subscription_plan" value="Monthly">
+          </div>
+          <div class="form-group">
+            <input type="checkbox" required id="accept" class="widget-form-checkbox" name="accept"> 
+            <p class="block-text form-text form-text-checkbox">I accept that Fresh Veggies saves my personal details</p>
```

Comment:
Overall a really good job with the indentations. Maybe just do the same here indentations here like you do further below.